channel/consensus_channel/consensus_channel.go

```diff
@@ -87,6 +87,13 @@ type Guarantee struct {
 	right  types.Destination
 }
 
+func (g Guarantee) Equal(g2 Guarantee) bool {
```

Comment:
1. Should this be a private method? (See https://github.com/statechannels/go-nitro/issues/408)
2. Should this be a pointer receiver? (See https://statechannels.slack.com/archives/C02CWRSH77X/p1648318876223959)
 
```suggestion
func (g *Guarantee) equal(g2 Guarantee) bool {
```

Reply:
The method is made private in c6a92a1. Note that the the other existing `Guarantee` method is value receiver https://github.com/statechannels/go-nitro/blob/c6a92a164efd947496b592c958c6e52109e7885f/channel/consensus_channel/consensus_channel.go#L98